iroh-resolver/src/content_loader.rs

```diff
@@ -39,6 +46,10 @@ impl<T: ContentLoader> ContentLoader for Arc<T> {
     async fn has_cid(&self, cid: &Cid) -> Result<bool> {
         self.as_ref().has_cid(cid).await
     }
+
+    async fn load_records_from_dht(&self, cid: &Cid) -> Result<iroh_rpc_types::p2p::PeerRecord> {
```

Comment:
Should be singular `record`, given it only returns a single record?

Reply:
Done